packages/cli/src/commands/lint.ts

```diff
@@ -108,3 +123,53 @@ export async function handleLint(
     process.exit(totals.errors === 0 || argv['generate-ignore-file'] ? 0 : 1),
   );
 }
+
+function lintConfigCallback(argv: LintOptions, version: string) {
+  if (argv['lint-config'] === 'off') {
+    return;
+  }
+
+  return async (config: RawConfig) => {
+    const configPath = findConfig(argv.config) || '';
+    const stringYaml = stringifyYaml(config);
+    const configContent = makeDocumentFromString(stringYaml, configPath);
+    const problems = await lintConfig({
+      document: configContent,
+      severity: argv['lint-config'] as ProblemSeverity,
+    });
+    await validateLintConfig({
+      problems,
+      outputFormat: argv.format,
+      maxProblems: argv['max-problems'],
+      version,
+    });
+  };
+}
+
+async function validateLintConfig({
```

Comment:
This function does not validate. It just prints the outputs. Please, rename.

Reply:
Do we need a function for this? maybe we can just inline it?

Reply:
Agree, this make sense for me!